test/frontend/mixins.spec.js

```diff
@@ -599,6 +599,71 @@ describe('Template Mixins', () => {
       });
     });
 
+    describe('maxlengthAttribute - input-text', () => {
```

Comment:
I think it might be safer to reset `render`, `render.should.have.been.calledWith` Otherwise, previous calls may interfere.
This might not work because I think `render.reset()` is in the older sinon versions before each test case
```suggestion
    describe('maxlengthAttribute - input-text', () => {
      beforeEach(() => {
      // Reset the render spy/mock before each test
      render.resetHistory();
  });
```

Reply:
Thanks Sulthan. since its a stub, I've used resetHistory now.